README.md

```diff
@@ -1,9 +1,38 @@
-# Pooled Cell Painting Experiment Repository Template
+# Pooled Cell Painting: Data Pipeline Welding Template :hammer_and_wrench:
 
-This repository was derived from a [template repository](https://github.blog/2019-06-06-generate-new-repositories-with-repository-templates/) located at https://github.com/broadinstitute/pooled-cp-profiling-template.
+This repository was derived from a [template repository](https://github.blog/2019-06-06-generate-new-repositories-with-repository-templates/) located at https://github.com/broadinstitute/pooled-cell-painting-profiling-template.
 The purpose of the repository is to weld together a versioned data processing pipeline with versioned processed output data for a single Pooled Cell Painting experiment.
 
-**AFTER GENERATING A NEW REPO, CHANGE OR DELETE ALL NONSPECIFIC DETAILS**
+## Setup computational environment
+
+First, install [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/).
+We use conda as an environment manager.
+
+```bash
+# Install computational environment
+conda env create --force --file environment.yml
+
+# Initialize the environment
+conda activate pooled-cell-painting
+```
+
+## Perform the weld
+
+The welding procedure is a three-step process.
+
+1. Activate conda environment (see above)
+2. Manually update the configuration yaml documents for your specific experiment
+    * Yaml documents with reasonable default values are available in the [config/](config/) folder.
+      * Do not change the location of these files.
+    * Additional documentation for each of the parameters is available in the [config/docs/](config/docs/) folder.
+3. Execute `weld.sh` (see below)
+
+```bash
+# After performing steps 1 and 2 above, perform step 3:
+./weld.sh
+```
+
+## **AFTER GENERATING A NEW REPO, CHANGE OR DELETE ALL NONSPECIFIC DETAILS**
```

Comment:
What is this line referring to? Where should someone change/delete things? What might they change/delete?

Reply:
Hmm, good point. The user should feel free to modify the README in any way. The README should reflect the specific experimental details and serve as a landing page to anyone viewing the dataset.

Therefore, I propose that we move all of this documentation and pictures to a new file called `SETUP.md` or `SETUP_INSTRUCTIONS.md`. This will keep the README minimalistic while still providing setup instructions (but not as the landing page).

We can then get rid of line 35

Reply:
To be more clear, move all lines below line 35

Reply:
For posterity, we just chatted and decided on the following plan:

*  I will address the figure suggestions, then request a re-review.
* Once approved, I will merge
* Then @ErinWeisbart will take over and take the README/SETUP documentation to the next level! 🚀